### PR TITLE
chore: fix more aria escaping edge cases

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: tests/page/to-match-aria-snapshot.spec.ts [hunk 1]
```diff
@@ -436,10 +436,16 @@ test('should unpack escaped names', async ({ page }) => {
 
   {
     await page.setContent(`
-      <button>Click \\ me</button>
+      <button>Click " me</button>
     `);
     await expect(page.locator('body')).toMatchAriaSnapshot(`
-      - button "Click \\ me"
+      - button "Click \\\" me"
```
Comment (yury-s) [inline tests/page/to-match-aria-snapshot.spec.ts:442]: add a test case with regex too?

**Review by yury-s — APPROVED** [review]

File: tests/page/to-match-aria-snapshot.spec.ts (hunk 1 above)
Comment (pavelfeldman) [reply tests/page/to-match-aria-snapshot.spec.ts:442]: Sure, I'll follow up.
